docs/docs/general/02-Getting Started/writing-your-first-test.md

```diff
@@ -19,7 +19,25 @@ This is going to create
   - `test/my-first-askui-test-suite.test.ts`: an example test with askui,
   - a folder `helper` which contains `jest.setup.ts` file: this file is setting up the test environment for all tests
   - a `jest.config.ts`: file for configuration of Jest,
-  
+
+### Configuration
+
+We need you to create some credentials through our [askui user portal](https://app.askui.com/) (usage is free!).
+Please see <a href="../askui%20User%20Portal/signup" target="_blank">our documentation on how to signup and create the credentials</a>.
+
+Then go to your `jest.config.ts` and add the configuration for your workspace Id and your access token to the _UiControlClient_.
+
+```typescript
+ aui = await UiControlClient.build({
+    credentials: {
+      workspaceId: 'workspace_id',
+      token: 'access token',
```

Comment:
Can we somehow highlight that `workspace_id` and `access token` should be replaced including making the casing consistent?

Reply:
Something like <your workspaceId> and <your access token>?